tests/snappi_tests/multidut/systest/test_over_subscription_imix.py

```diff
@@ -0,0 +1,515 @@
+    # With imix flag set, the traffic_generation.py uses IMIX profile.
+    pkt_size = 1024
+
+    # Percentage drop expected for lossless and lossy traffic.
+    # speed_tol is speed tolerance between egress link speed and actual speed.
+    # loss_expected to check losses on DUT and TGEN.
+    test_check = {'lossless': 0, 'lossy': 0, 'speed_tol': 14, 'loss_expected': False, 'pfc': False}
+
+    test_def = {'TEST_FLOW_AGGR_RATE_PERCENT': 18,
+                'BG_FLOW_AGGR_RATE_PERCENT': 27,
+                'data_flow_pkt_size': pkt_size,
+                'DATA_FLOW_DURATION_SEC': 300,
+                'data_flow_delay_sec': 0,
+                'SNAPPI_POLL_DELAY_SEC': 60,
+                'test_type': '/tmp/Two_Ingress_Single_Egress_non_cngstn_'+str(port_map[1])+'Gbps',
+                'line_card_choice': line_card_choice,
+                'port_map': port_map,
+                'enable_pfcwd': True,
+                'enable_credit_wd': True,
+                'stats_interval': 60,
+                'background_traffic': True,
+                'imix': False,
+                'test_check': test_check,
+                'verify_flows': False}
+
+    logger.info('Starting the test for line_card_choice : {}'.format(line_card_choice))
+    if line_card_choice not in linecard_configuration_set.keys():
+        assert False, "Invalid line_card_choice value passed in parameter"
+
+    if (len(linecard_configuration_set[line_card_choice]['hostname']) >= 2):
+        dut_list = [dut for dut in duthosts
+                    if dut.hostname in linecard_configuration_set[line_card_choice]['hostname']]
+        duthost1, duthost2 = dut_list
+    elif (len(linecard_configuration_set[line_card_choice]['hostname']) == 1):
+        dut_list = [dut for dut in duthosts
+                    if linecard_configuration_set[line_card_choice]['hostname'] == [dut.hostname]]
+        duthost1 = duthost2 = dut_list[0]
+    else:
+        assert False, "Hostname can't be an empty list"
+
+    snappi_port_list = get_multidut_snappi_ports(line_card_choice=line_card_choice,
+                                                 line_card_info=linecard_configuration_set[line_card_choice])
+
+    if len(snappi_port_list) < 3:
+        assert False, "snappi_port_list: Need Minimum of 3 ports for the test"
+
+    # port_map is read as egress port_map[0] links of port_map[1] speed
+    # and ingress port_map[2] links of port_map[3] speed
+    port_map = test_def['port_map']
+
+    snappi_ports = new_get_multidut_tgen_peer_port_set(line_card_choice, snappi_port_list, config_set, port_map)
+
+    if len(snappi_ports) < 3:
+        assert False, "snappi_ports: Need Minimum of 3 ports for the test"
+
+    testbed_config, port_config_list, snappi_ports = snappi_dut_base_config(dut_list,
+                                                                            snappi_ports,
+                                                                            snappi_api)
+
+    test_prio_list = [3, 4]
```

Comment:
can be replaced by fixture lossless_prio_list, and bg_prio_list can be replaced by fixture lossy_prio_list

Reply:
This is replaced by fixture lossless_prio_list and bg_prio_list.